### ISSUE TITLE
Keyboard scrolling does not work in window contents (minor)

### ISSUE THREAD

**Issue by ttechnicus** [acdh-oeaw/vicav-vue3#95]
How to reproduce:
- open a window with long enough content (e. g. text) to display vertical scrollbar
- click inside the contents to ensure focus
- try to scroll the contents with the up/down arrow keys on the keyboard

Expected:
- the contents scroll

Currently:
- no scrolling happens
